ui/dashboards/src/test/testDashboard.ts

```diff
@@ -28,108 +28,163 @@ const testDashboard: DashboardResource = {
     duration: '24h',
     variables: [
```

Comment:
Just seeing that, sorry for the late.

Wouldn't be easier to put the variables as a map like the panels. So it's easier to reference it.
That would be consistent also.
@saminzadeh @LukeTillman @sjcobb

Reply:
If we did / do ever go with second approach, maybe we'd want something like:
```
variables: { ... },
variableLayout: [...],
panels: { ... },
panelLayouts: [ ... ],
```

Basically, prefixing `layouts` so that one is for panels and one is for variables

Reply:
Let's definitely follow-up on this if you all think it merits more discussion. We can definitely do it in a follow-up PR if we feel like we need to.

Reply:
maybe managing a layout for the variable might be painful for the users. I'm quite lazy, so the less I have to do to configure my dashboard, the better I would be 🤣